src/models/auth.py

```diff
@@ -1,10 +1,15 @@
+import os
 from datetime import datetime
 from time import time
 from uuid import uuid4
 from . import db
 from .base import BaseModel
+from main.config import config_by_name
 
-CONFIRMATION_EXPIRATION_DELTA = 60 * 30  # 60seg * 30min = 1800 
+config = config_by_name[os.getenv('APP_ENV') or 'dev']
+
+                            # 60seg*36min*24h = 1d 
+CONFIRMATION_EXPIRATION_DELTA = 60*60*24*int(config.ACCOUNT_EMAIL_CONFIRMATION_EXPIRE_DAYS)
```

Comment:
you can use timedelta here so you avoid doing multiplication. https://docs.python.org/3/library/datetime.html#datetime.timedelta